input/pagecontent/terminology.md

```diff
@@ -37,6 +37,7 @@ Many value sets used in this guide are defined in the base FHIR specification, [
 |[RequestIntent](https://hl7.org/fhir/R4/valueset-request-intent.html)|[AU eRequesting ServiceRequest](StructureDefinition-au-erequesting-servicerequest.html)|FHIR|
 |[RequestStatus](https://hl7.org/fhir/R4/valueset-request-status.html)|[AU eRequesting ServiceRequest](StructureDefinition-au-erequesting-servicerequest.html)|FHIR|
 |[AU eRequesting Task Status](ValueSet-au-erequesting-task-status.html)|[AU eRequesting Task](StructureDefinition-au-erequesting-task.html)|AU eRequesting|
+|[AU eRequesting Coverage Type Self-pay](ValueSet-au-erequesting-coverage-self-pay.html)||AU eRequesting|
```

Comment:
Please update Code systems table to add code systems referenced by the new value set.

Reply:
I've added the 2 missing CodeSystems to the table in section 8.2